packages/manager/modules/pci/src/projects/project/ai-dashboard/components/ai-pipeline/ai-pipeline.html

```diff
@@ -0,0 +1,140 @@
+<div class="row align-items-flex-start">
+    <div class="col">
+        <div class="d-flex flex-column align-items-center">
+            <img data-ng-src="{{::$ctrl.images.stock}}" alt="stock" />
+            <strong>
+                <span>1.</span>
+                <span data-translate="pci_ai_dashboard_pipeline_stock"></span>
+            </strong>
```

Comment:
i guess it's better to use native numeration list.
`<ol>
  <li data-translate="pci_ai_dashboard_pipeline_stock" class="font-weight-bold"></li>
</ol>`**

Reply:
Good idea; changed here: https://github.com/ovh/manager/pull/8146/commits/33d7ae3ff07390132b56538adf2cb6b8bcd22adc